src/PSServiceBus/Cmdlets/ReceiveSbMessage.cs

```diff
@@ -9,7 +9,7 @@ namespace PSServiceBus.Cmdlets
     /// <summary>
     /// <para type="synopsis">Receives a message or messages from an Azure Service Bus queue or subscription.</para>
     /// <para type="description">This cmdlet retrieves a message or messages from an Azure Service Bus queue or subscription.  Two receive modes are available:</para>
-    /// <para type="description">ReceiveAndKeep (default) and ReceiveAndDelete which if specified will remove the message from the queue.  Multiple messages can be</para>
+    /// <para type="description">ReceiveAndKeep/PeekOnly (default) and ReceiveAndDelete which if specified will remove the message from the queue.  Multiple messages can be</para>
```

Comment:
A couple of small things I think we should look to do here:

1. Change the default value of the [ReceiveType parameter](https://github.com/tommagumma/PSServiceBus/pull/2/files#diff-9210e7fb022394286ff91a7b6488a9d1L82) to use the new `SbReceiveTypes` member

2. Issue a warning to the user if they use the old `ReceiveAndKeep` option telling them that it will be deprecated in future versions - adding something like this would do it:

```
if (this.ReceiveType == SbReceiveTypes.ReceiveAndKeep)
{
    WriteWarning("The option ReceiveAndKeep will be deprecated in future versions.  Please use 'PeekOnly' instead.");
}
```